src/lib.rs

```diff
@@ -149,17 +149,15 @@
 //! However, external RNGs can fail, and being able to handle this is important.
 //!
 //! It has therefore been decided that *most* methods should not return a
-//! `Result` type, and must therefore panic if an error does occur. The
-//! exceptions are [`Rng::try_fill`], [`RngCore::try_fill_bytes`], and
-//! [`SeedableRng::from_rng`].
+//! `Result` type, with as exceptions [`Rng::try_fill`],
+//! [`RngCore::try_fill_bytes`], and [`SeedableRng::from_rng`].
+//!
+//! Note that it is the RNG that panics when it fails but is not used through a
+//! method that can report errors. Currently Rand contains only three RNGs that
+//! can return an error (and thus may panic), and documents this property:
+//! [`OsRng`], [`EntropyRng`] and [`ReadRng`]. Other RNGs, like [`ThreadRng`]
+//! and [`StdRng`], can be used with all methods without concern.
```

Comment:
ThreadRng can panic on initialisation.

(Can I still write that the English way?)